eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -93,10 +93,19 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 		bodyString := strings.ReplaceAll(string(body), "\n", "\n    ")
 		dataMap[path.Base(uri)] = bodyString
 	}
+	if len(c.Config.Spec.ClusterLoader.TestOverrides) > 0 {
+		overrides := strings.Join(c.Config.Spec.ClusterLoader.TestOverrides, "\n    ")
+		overrides = strings.ReplaceAll(overrides, "=", ": ")
+		dataMap["overrides.yaml"] = overrides
```

Comment:
Is there any harm in letting dataMap[overrides.yaml] = "". That could simplify your if check.

Reply:
I don't believe so, if there isn't harm, I'll change it.

Reply:
Returns an error, keeping the check.